<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="" tests="1" assertions="1" errors="0" warnings="0" failures="1" skipped="0" time="3.800139">
    <testsuite name="Feature" tests="1" assertions="1" errors="0" warnings="0" failures="1" skipped="0" time="3.800139">
      <testsuite name="Tests\Feature\ProdutoTest" file="/home/vagrant/tdd/tests/Feature/ProdutoTest.php" tests="1" assertions="1" errors="0" warnings="0" failures="1" skipped="0" time="3.800139">
        <testcase name="um_produto_pode_ser_criado" class="Tests\Feature\ProdutoTest" classname="Tests.Feature.ProdutoTest" file="/home/vagrant/tdd/tests/Feature/ProdutoTest.php" line="18" assertions="1" time="3.800139">
          <failure type="PHPUnit\Framework\ExpectationFailedException">Tests\Feature\ProdutoTest::um_produto_pode_ser_criado
Failed asserting that a row in the table [produtos] matches the attributes {
    "nome": "Ipsam",
    "descricao": "Nihil deleniti temporibus exercitationem voluptatum blanditiis esse.",
    "lance_minimo": 266796.7,
    "ordem": 106,
    "valor_buyout": 2451861.673
}.

The table is empty..

/home/vagrant/tdd/vendor/laravel/framework/src/Illuminate/Foundation/Testing/Concerns/InteractsWithDatabase.php:28
/home/vagrant/tdd/tests/Feature/ProdutoTest.php:31</failure>
        </testcase>
      </testsuite>
    </testsuite>
  </testsuite>
</testsuites>
